implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -356,15 +361,16 @@ impl NodeManager {
                 Some(f) => BootstrapedIdentityStore::new(Arc::new(f), repository.clone()),
             });
 
+        info!("create the secure channels service");
         let secure_channels = SecureChannels::builder()
             .with_identities_vault(vault)
             .with_identities_repository(identities_repository.clone())
             .build();
 
         let policies: Arc<dyn PolicyStorage> = Arc::new(node_state.policies_storage().await?);
 
-        let medic = Medic::new();
-        let sessions = medic.sessions();
+        info!("NodeManager::create: start the Medic");
```

Comment:
This one should also have the `NodeManager::create:` removed right?

Reply:
I missed that one, thanks